xbzu1/1.0/board.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?> 
+<!DOCTYPE board SYSTEM "C:/Xilinx/Vivado/2021.1/data/boards/board_schemas/current/board.dtd">
```

Comment:
We should be able to use an env var to substitute the Vivado path.  Like this:
"{$XILINX_VIVADO}/data/boards/board_schemas/current/board.dtd"